container-host-files/etc/hcf/config/scripts/configure-HA-hosts.sh

```diff
@@ -53,6 +56,9 @@ if test -z "${KUBE_SERVICE_DOMAIN_SUFFIX:-}" ; then
     # bootstrap node.
     export KUBE_SERVICE_DOMAIN_SUFFIX="$(awk '/^search/ { print $2 }' /etc/resolv.conf)"
 fi
+
+export KUBE_COMPONENT_INDEX="$(cat /var/vcap/instance/id)"
```

Comment:
@mook-as i think we should expose the ID as an env var [here](https://github.com/SUSE/fissile/blob/9ca01d6a6bd6ab9ddab78e112506ecdc0b592c8e/scripts/dockerfiles/run.sh#L85) and use it before merging.
Also, the pieces that are not `scf` specific in this file should move to `run.sh`

Reply:
Sure, I can do that.